collector/benchmarks/README.md

```diff
@@ -246,3 +246,51 @@ Rust code being written today.
 - Compare the benchmarking time of the two versions.
 - When adding the new version, for `perf-config.json` and the `N-*.patch`
   files, use the corresponding files for the old version as a starting point.
+
```

Comment:
I'm not sure how others think, but I would like to avoid the README from getting to long. I personally prefer shorter READMEs that link to other documentation. I find it makes finding information much easier when you don't have to scroll and search manually for where something might live.

Reply:
I like having it all in one place :)  Would a table of contents help? Can markdown autogenerate one of those?

Reply:
A table of contents would certainly help. I already know where info is in this repo's README files, and I already find it difficult to find things because they are just walls of text. As for automation, there are tools like [markdown-toc](https://github.com/jonschlinkert/markdown-toc), but I don't have much experience with them.